app/views/api/integrations/apicast/configuration_driven/_apicast.html.slim

```diff
@@ -54,6 +54,7 @@ section.Configuration class=('Configuration--is-promoted' unless deployment_opti
           dl.u-dl
             dt.u-dl-term URL
             dd.u-dl-definition = @show_presenter.staging_proxy_endpoint
+          - unless @proxy.service.oauth?
             dt.u-dl-term Example curl for testing
```

Comment:
You need to indent the below section to have it taken into account

Reply:
Done